goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["pov-camera",                                 "pov-camera",                 3, ["ENGINE", "GAME"], "engine/camera"],
+["powerups",                                   "powerups",                   3, ["ENGINE", "GAME"], "engine/game"],
+["crates",                                     "crates",                     3, ["ENGINE", "GAME"], "engine/game"],
+["hud",                                        "hud",                        3, ["ENGINE", "GAME"], "engine/ui"],
+["hud-classes",                                "hud-classes",                3, ["ENGINE", "GAME"], "engine/ui"],
+["progress-static",                            "progress-static",            3, ["ENGINE", "GAME"], "engine/ui/progress"],
+["progress",                                   "progress",                   3, ["ENGINE", "GAME"], "engine/ui/progress"],
+["progress-draw",                              "progress-draw",              3, ["ENGINE", "GAME"], "engine/ui/progress"],
+["ocean",                                      "ocean",                      3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-vu0",                                  "ocean-vu0",                  3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-texture",                              "ocean-texture",              3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-mid",                                  "ocean-mid",                  3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-transition",                           "ocean-transition",           3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-near",                                 "ocean-near",                 3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["minimap",                                    "minimap",                    3, ["ENGINE", "GAME"], "engine/ui"],
+["bigmap-data",                                "bigmap-data",                3, ["ENGINE", "GAME"], "engine/ui"],
+["bigmap",                                     "bigmap",                     3, ["ENGINE", "GAME"], "engine/ui"],
+["eye",                                        "eye",                        3, ["ENGINE", "GAME"], "engine/gfx"],
+["glist-h",                                    "glist-h",                    3, ["ENGINE", "GAME"], "engine/util"],
+["glist",                                      "glist",                      3, ["ENGINE", "GAME"], "engine/util"],
+["anim-tester",                                "anim-tester",                3, ["ENGINE", "GAME"], "engine/debug"],
+["viewer",                                     "viewer",                     3, ["ENGINE", "GAME"], "engine/debug"],
+["part-tester",                                "part-tester",                3, ["ENGINE", "GAME"], "engine/debug"],
+["editable-h",                                 "editable-h",                 3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["editable",                                   "editable",                   3, ["ENGINE", "GAME"], "engine/debug/nav"],
```

Comment:
I think this can edit a lot more, so maybe it belongs in `engine/debug`. It looks like it can edit regions, which control camera, lighting, and other things.